README.md

```diff
@@ -391,6 +391,16 @@ The configuration above does the following:
 *   Proxy requests with path `/api/v1/proxy/healthplan/some/mock/path` to the mock server. Optional configuration that is useful if an API is not available for use and needs to be mocked.
 *   Proxy all requests with path segments `/api/v1/proxy/healthplan/` to the configured target `'http://localhost:8888'`. Notice the URL is being rewritten. Change the rewrite path to match your local path as needed. This configuration is useful when testing against live services.
 
+### Tired of deeploy nested components?
```

Comment:
Is this feature dependent on a specific version of Availity-Workflow?
If so, consider changing the header to
```
### Tired of deeploy nested components? (As of version #.#)
```

Not asking for a change just trying to spark some discussion. 👍

Reply:
I agree there definitely should be docs around this feature. Currently the developer of that import plugin doesn't want to accept the current PR for relative imports with `@`. I may have to close this and re-open once we either submit a new PR with that feature or someone else does to that repo.